hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/PlanExecutor.java

```diff
@@ -683,7 +684,7 @@ SqlResult execute(IMapUpdatePlan plan, List<Object> arguments, long timeout, @No
 
         Object key = plan.keyCondition().eval(EmptyRow.INSTANCE, evalContext);
         CompletableFuture<Long> future = hazelcastInstance.getMap(plan.mapName())
-                .submitToKey(key, plan.updaterSupplier().get(evalContext))
+                .submitToKey(key, plan.updaterSupplier().get(UntrustedExpressionEvalContext.from(evalContext)))
```

Comment:
`UntrustedExpressionEvalContext` is needed only after deserialization. Or is used here for consistency for local execution when there is no serialization?

Reply:
e00f44dab5e6ace1fe6b3f686f35fee517090bb7